### PR TITLE
WRO-13139: Sampler: Remove controls from Docs tab

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In storybook, if controls are changed from Docs tab, it doesn't update the component preview. 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Setting "inlineStories" to true causes bugs, so we agreed to keep inlineStories false and set a height to the iFrame. Also, controls were removed from the docs page.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
WRO-13139

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM but let's see SWP team opinion

**Comment by adrian-cocoara-lgp** [comment]
Travis fail because of node v19

File: samples/sampler/stories/default/About.js [hunk 1]
```diff
@@ -64,7 +65,17 @@ const HintDialog = kind({
 });
 
 export default {
-	title: 'About/A Tour of Sampler'
+	title: 'About/A Tour of Sampler',
+	parameters: {
+		docs: {
```
Comment (MikyungKim) [inline samples/sampler/stories/default/About.js:70]: Oh... you don't need to do this for every story. You could just do it from the preview.js.

Comment (stanca-pop-lgp) [reply samples/sampler/stories/default/About.js:70]: Oh, I don't know why I didn't think of that. I will fix this. Thank you, Mikyung!

Comment (stanca-pop-lgp) [reply samples/sampler/stories/default/About.js]: I kept the imports sorting if that is ok.

File: samples/sampler/.qa-storybook/preview.js [hunk 2]
```diff
@@ -48,12 +49,19 @@ const skins = {
...
 
 configureActions();
+
 export const parameters = {
 	docs: {
 		container: DocsContainer,
-		page: DocsPage,
-		iframeHeight: 360,
 		inlineStories: false,
+		iframeHeight: ri.scaleToRem(300),
```
Comment (MikyungKim) [inline samples/sampler/.qa-storybook/preview.js:57]: How about 600? It seems too small.

File: samples/sampler/.storybook/preview.js [hunk 3]
```diff
@@ -47,12 +48,18 @@ const skins = {
...
 
 configureActions();
+
 export const parameters = {
 	docs: {
 		container: DocsContainer,
-		page: DocsPage,
-		iframeHeight: 360,
 		inlineStories: false,
+		iframeHeight: ri.scaleToRem(300),
```
Comment (MikyungKim) [inline samples/sampler/.storybook/preview.js:56]: How about 900? It seems too small.

**Pull request merged**